Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -2,7 +2,7 @@ commonfields:
   id: Recorded Future v2
   version: -1
 name: Recorded Future v2
-display: Recorded Future v2
+display: Recorded Future v2.1
```

Comment:
@jonathan-s why have we decided to do this change? The `v2` refers to the difference between the previous (XSOAR supported) pack to this one. I think the addition `.1` could confuse customers with the real pack version (which shouldn't appear in the display name).